app/models/solr_document.rb

```diff
@@ -120,6 +120,18 @@ def file_ids
     self[:img_info] || self[:file_id] || []
   end
 
+  def location_display
+    @location = holdings.libraries&.first
```

Comment:
Instead of an instance variable here, can you use a local?  How about using `library` instead as there are `Locations` within a library, so using the name `location` could be confusing.   Would it makes more sense to look at `folio_holdings` here as `holdings` also has `mhlds`

Reply:
Where are you thinking we should using folio_holdings? In this line and has_physical_copies or just this line?